common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1438,7 +1408,7 @@ def forum_posts_allowed(self):
 
     @property
     def number(self):
```

Comment:
I now see why you called it a course number. Perhaps a docstring might be helpful here to explain what a "number" is and then changing the helper method number_for_course_location to be more representative of what a course "number" really is.